packages/in_app_purchase/in_app_purchase/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: in_app_purchase
 description: A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store and Google Play.
 repository: https://github.com/flutter/packages/tree/main/packages/in_app_purchase/in_app_purchase
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+in_app_purchase%22
-version: 3.1.13
+version: 3.2.0
 
 environment:
   sdk: ^3.1.0
```

Comment:
FYI these values will need to be updated to match `i_a_p_storekit`'s lower limits when you do the final app-facing PR, otherwise `pub get` will fail.